src/Sentry/TransactionTracer.cs

```diff
@@ -278,6 +310,18 @@ private void AddChildSpan(SpanTracer span)
     /// <inheritdoc />
     public void Finish()
     {
+        if (Interlocked.Exchange(ref _idleTimerStopped, 1) == 0)
+        {
+            _idleTimer?.Change(Timeout.InfiniteTimeSpan, Timeout.InfiniteTimeSpan);
```

Comment:
debug log here too is a good idea

Reply:
See https://github.com/getsentry/sentry-dotnet/pull/2547